src/doom/d_main.c

```diff
@@ -1126,6 +1126,12 @@ static void InitGameVersion(void)
         deathmatch = 1;
     }
     
+    // -fast was not an option until Doom v1.4
+    if (gameversion <= exe_doom_1_2)
+    {
+        fastparm = 0;
```

Comment:
I am also in favor of a warning message printed to `stdout` here. Sure, Vanilla Doom didn't write a lot. But Vanilla Doom didn't try to emulate the behaviour of previous releases of itself either. And I find it confusing enough if a parameter does work for one emulated version of Doom but not for the other, so a hint for the concerned user is very much justified.